### PR TITLE
update student overview query to account for timezones and test

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Update the query that shows on the Student Overview page under Activity Scores to adjust the `completed_at` date for the teacher's timezone.

## WHY
A teacher pointed out that they saw a discrepancy between these dates and what shows on their Activity Summary, which shouldn't be the case.

## HOW
Pass `current_user` to the `student_overview` query and use it to adjust the `completed_at` date that gets sent to the frontend, as we do with other queries.

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
https://www.notion.so/quill/Student-s-Activity-Summary-dates-do-not-match-the-student-s-Activity-Score-dates-13ed42e6f941804d8a50e045001891d1?d=13ed42e6f94180ec85ae001cca79f595

### What have you done to QA this feature?
Locally tested that timezones are now taken into account on this page.

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | YES
Have you deployed to Staging? | NO - tiny change
Self-Review: Have you done an initial self-review of the code below on Github? | YES

### CODE REVIEW

File: services/QuillLMS/spec/queries/progress_reports/student_overview_spec.rb
Comment (brendanshean) [inline services/QuillLMS/spec/queries/progress_reports/student_overview_spec.rb:1]: I appreciate the shout-out to Chicago.

**Review by brendanshean — APPROVED** [review]
Thank you for fixing this

**Review by eadams17 — APPROVED** [review]
Thanks for fixing this!

**Review by anathomical — APPROVED** [review]
Definitely beyond the scope of this PR, but it feels like this is probably something we should standardize an approach for at some point.  I know we do this specific thing in a couple of places, but there's probably some cleaner thing we could be doing more generally.

Not a high priority, but something sitting in the back of my mind now.

File: services/QuillLMS/spec/queries/progress_reports/student_overview_spec.rb
Comment (emilia-friedberg) [reply services/QuillLMS/spec/queries/progress_reports/student_overview_spec.rb:1]: @brendanshean unfortunately, 'America/Philadelphia' is not a timezone in our system.

Comment (brendanshean) [reply services/QuillLMS/spec/queries/progress_reports/student_overview_spec.rb:1]: Well, at least not _yet_. 🔔

**Pull request merged**